orchestration/api/api_rank_active_learning.py

```
@@ -424,6 +426,28 @@ async def add_datapoints(request: Request, selection: RankSelection, image_sourc
         dict_data['image_1_metadata']['image_source'] = image_source
         dict_data['image_2_metadata']['image_source'] = image_source
 
+        # Get image_uuid for image_1_metadata
+        image_1_hash = dict_data['image_1_metadata']['file_hash']
+        image_1_uuid = request.app.all_image_collection.find_one(
+            {"image_hash": image_1_hash, "bucket_id": get_bucket_id(image_source)},
+            {"uuid": 1}
+        )
+        if image_1_uuid:
+            dict_data['image_1_metadata']['image_uuid'] = image_1_uuid['uuid']
+        else:
+            print(f"Image UUID not found for image_1_metadata with hash: {image_1_hash}")
```

If image uuids are going to be mandatory now, the funstion should fail if the value is not found, to avoid leaving incomplete data in the database. This should be also done when checking the uuid from the second image, and for both images in "/rank-training/add-ranking-data-point-v1".